src/main/java/Translator.java

```diff
@@ -105,7 +119,7 @@ protected void processLine(String line) {
         Scale scale = new Scale();
         for (int k = 0; k < itemCount; k++) {
             String label = row[k];
-            int value = Integer.parseInt(row[k + itemCount]);
+            CellContent value = new CellContent(row[k + itemCount]);
```

Comment:
Is there a possibility that there might happen an invalid indexing (i.e. is there some kind of guard index validation required or are we good to go?)